### PR TITLE
Style edits

### PR DESCRIPTION
**Pull request by awfrancisco**

### CODE REVIEW

File: pages/who-we-are-hiring.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ Everywhere in the United States! If you have no experience working on remote tea
 We're actively hiring for the following positions:
 
 ###Software development (front end)
-Using modern, open source web frameworks and tools, our front and back end developers work in small, cross-functional teams, close to user research and product prioritization, to produce functional, scalable web services.
```
Comment (fureigh) [inline pages/who-we-are-hiring.md:23]: @awfrancisco Thanks for the edits! Can you share a little more context about this one?

Comment (fureigh) [inline pages/who-we-are-hiring.md:23]: I'm still curious, but looks like there was a merge jinx w/ @emileighoutlaw. :)

Comment (emileighoutlaw) [inline pages/who-we-are-hiring.md:23]: Yeah, I'm really sorry!

Comment (awfrancisco) [inline pages/who-we-are-hiring.md:23]: @fureigh The edit was for readability. The original sentence had too many commas and clauses to be easily readable, and we should avoid jargony words like prioritization. If it's important to mention how the team relies on user research and has a hand in product prioritization, I think that should be moved to another sentence.

Comment (fureigh) [inline pages/who-we-are-hiring.md:23]: @emileighoutlaw: No worries.

@awfrancisco: Thanks for the context. Personally it made a difference to me to see that developers are close to user research and product prioritization, so I may take a shot at reincorporating that in another sentence (or bounce this to one of our new content folks).

Comment (awfrancisco) [inline pages/who-we-are-hiring.md:23]: Sounds good!

**Pull request merged**